common/lib/aws_assume_role.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python3
```

Comment:
can we move the `common/lib/` dir to `common/testdata/lib/`, so that it's guaranteed to be ignored by the go tooling?